ModerationPolicy.md

```diff
@@ -0,0 +1,356 @@
+default options are 24-hour, 48-hour, and 7-day periods.
+
+An Indefinite Block is set for an unspecified period of time and may only be
+lifted for an individual through a simple majority vote of the Moderation
+Team.
+
+## Privacy of the expressjs/moderation Repository
+
+The expressjs/moderation Repository is used to discuss the potentially sensitive
+details of any specific moderation issue. The repository is private but
+accessible to all Collaborators. The details of any issue discussed within the
+expressjs/moderation repository are expected to remain confidential and are not to
+be discussed in any public forum or social media service.
+
+Any Collaborator found to be violating the privacy of the expressjs/moderation
+repository by repeatedly sharing or discussing the details of expressjs/moderation
+issues in any public forum or social media service risks being permanently
+removed from the expressjs GitHub organization.
+
+## Moderation Team
+
+The expressjs Moderation Team is tasked with enforcement of this policy.
+
+At least once per month, the Moderation Team must provide a report of all Moderation
+actions taken by the Moderation Team to the TC.
+
+_Requirements_
+
+-   Must be a Collaborator in the expressjs GitHub organization.
+-   Active participation in the expressjs/moderation repository.
+-   Show a commitment to the expressjs [Code of Conduct][].
+
+_Nomination_
+
+Moderation team members are Collaborators who self-nominate or are nominated by
+the TC. Team members must be approved by the TC with annual
+recertification. If there are no objections after seven days, the nomination is automatically
+accepted. If there are objections to a specific nomination, then a TC vote
+in favor of the nomination is required.
+
+_Onboarding_
+
+New Moderation Team members are onboarded with:
+
+-   an invite to expressjs Moderation Team channel in the OpenJS Slack
+-   permission changes made to GitHub to allow access to moderate
+-   a walkthrough of relevant processes, expectations, and documents by an existing Moderation Team member
+-   access to existing documents
+
+_Recertification_
+
+An annual recertification vote is required for all Moderation Team members.
+For an individual to be recertified, a TC vote in favor of recertification is required.
+
+_Departure_
+
+A TC vote is required to remove a moderator who has not resigned.
+
+_Resignation_
+
+At any time a Moderation Team member may notify the team that they will no longer be serving.
+Either the resigning member or an active member will file an issue notifying the Admin group
+that the team member is stepping down. An active team member will take necessary steps to
+remove resigning team member from respective permissions and private access.
+
+<!-- referenced from the CoC page -->
+
+<a id="current-members"></a>
+
+### Current Members of Moderation Team
+
+-   [IamLizu](https://github.com/IamLizu) -
```

Comment:
initial members should probably be empty and go through the documented process

Reply:
Sure, purging the list.